packages/shared/package.json

```diff
@@ -21,7 +21,7 @@
     "@chainflip/bitcoin": "^1.2.4",
     "@chainflip/rpc": "^1.8.9",
     "@chainflip/solana": "^2.1.4",
-    "@chainflip/utils": "^0.5.0",
+    "@chainflip/utils": "^0.8.0",
```

Comment:
we can probably move the @chainflip packages to the root package.json ?

Reply:
we can't here because we publish packages and they need to all be listed in the `package.json` as dependencies